### PR TITLE
ZK-4628: Toggle Popup with onOpen doesn't close

### PR DESCRIPTION
**Pull request by plijyfes**

### CODE REVIEW

File: zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4628Test.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import org.junit.Assert;
+import org.junit.Test;
+import org.zkoss.zktest.zats.WebDriverTestCase;
+import org.zkoss.zktest.zats.ztl.JQuery;
+
+public class B95_ZK_4628Test extends WebDriverTestCase {
+	@Test
+	public void test() {
+		connect();
+		testTogglePopup(0); // test with VM onOpen listener
```
Comment (scribetw) [inline zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4628Test.java:23]: testTogglePopup(0, true)

**Review by scribetw — CHANGES_REQUESTED** [review]
case refine

**Review by DevChu — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**